src/main/java/com/jwplayer/jwplatform/JWPlatformClient.java

```diff
@@ -63,6 +67,21 @@ public static JWPlatformClient create(final String apiKey, final String apiSecre
     return new JWPlatformClient(apiKey, apiSecret);
   }
 
+  /**
+   * Instantiate a new {@code JWPlatformClient} instance.
+   *
+   * @param apiSecret - your api key
+   * @param apiKey - your api secret
+   * @return a {@code JWPlatformClient} instance
+   */
+  public static JWPlatformClient create(final String apiKey, final String apiSecret, final String host) {
```

Comment:
Layer the other create().

```
public static JWPlatformClient create(final String apiKey, final String apiSecret) {
   return create(apiKey, apiSecret ,"https://api.jwplatform.com/v1/");
}
```

Then you can get rid of the constructor: `private JWPlatformClient(final String apiKey, final String apiSecret)`